Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py

```diff
@@ -341,16 +342,19 @@ def list_members_command(client: MsGraphClient, args: Dict) -> Tuple[str, Dict,
 
     # get the group data from the context
     group_data = demisto.dt(demisto.context(), f'{INTEGRATION_CONTEXT_NAME}(val.ID === "{group_id}")')
+    if not group_data:
+        return_error('Could not find group data in the context, please run "!msgraph-groups-get-group" to retrieve group data.')
     if isinstance(group_data, list):
         group_data = group_data[0]
 
     if '@odata.nextLink' in members:
         next_link_response = members['@odata.nextLink']
         group_data['Members'] = members_outputs  # add a field with the members to the group
-        group_data['Members']['NextLink'] = next_link_response
+        group_data['MembersNextLink'] = next_link_response
```

Comment:
Please update the command's outputs for both context paths you changes/added

Reply:
Done